articles/libraries/_includes/_configure_embedded_login.md

```diff
@@ -0,0 +1,13 @@
+### Configure Auth0 for Embedded Login
+
+<% if (library == 'lock') { %>
```

Comment:
Is there a reason to separate these out with a conditional when they say practically the same thing? Could it be something like this?

```
Cross-origin calls inside hidden iframes will be used to perform authentication. To make sure this can be done securely, Auth0 needs to know the domains where you will be hosting your applications.
```